docs/en/01_configuration.md

```diff
@@ -143,71 +150,72 @@ environment. In this environment, RealMe provide all SSL certificates required t
 - Download the integration bundle from the [RealMe Developers site](https://developers.realme.govt.nz/try-it-out-now/).
 - Unpack the following three certificates into the directory you've specified in `REALME_CERT_DIR` (outside of your webroot):
-    - `mts_assert_saml_idp.cer`
-    - `mts_login_saml_idp.cer`
-    - `mts_saml_sp.pem`
+  - `mts_assert_saml_idp.cer`
+  - `mts_login_saml_idp.cer`
+  - `mts_saml_sp.pem`
 - The `mts_assert_saml_idp.cer` and `mts_login_saml_idp.cer` files are not correctly provided. You will need to manually add the following to the files:
-    - Add a new line as line 1 of the file with the following: `-----BEGIN CERTIFICATE-----`
-    - Add a new line as the last line of the file with the following: `-----END CERTIFICATE-----`
+  - Add a new line as line 1 of the file with the following: `-----BEGIN CERTIFICATE-----`
+  - Add a new line as the last line of the file with the following: `-----END CERTIFICATE-----`
 - Ensure your `realme.yml` [configuration](docs/en/configuration.md) is complete (see above).
 - Run the RealMe build task to validate your configuration and get the XML metadata to provide to MTS: `vendor/bin/sake dev/tasks/RealMeSetupTask forEnv=mts`
 - Save the XML output from the above task to an XML file, and upload this to MTS:
-    - For a 'logon' integration, submit here: [MTS logon metadata upload](https://mtscloud.realme.govt.nz/Login/home).
-    - For an 'assert' integration, submit here: [MTS assert metadata upload](https://mtscloud.realme.govt.nz/Assertion/home).
+  - For a 'logon' integration, submit here: [MTS logon metadata upload](https://mtscloud.realme.govt.nz/Login/home).
+  - For an 'assert' integration, submit here: [MTS assert metadata upload](https://mtscloud.realme.govt.nz/Assertion/home).
 - Either use the `$RealMeLoginForm` global template variable or add the `RealMeAuthenticator` and access `/Security/login`.
 - Once authenticated, you can access user data from templates using `$RealMeUser` (e.g. `$RealMeUser.SPNameID`), or in a controller by using `RealMeService::currentRealMeUser()`.
 
 If you are developing locally, note that the module enforces your environment to be configured for https. If you don't
 have this setup by default, [ngrok](https://ngrok.com/download) is a nice, easy to use tool that provides this
 functionality. You just run ngrok, and copy the https URL that it gives you - this will let you access your site
-protected via https, however you will need to ensure you set the `SS_TRUSTED_PROXY_IPS` const in your
-_ss_environment.php, e.g. `define('SS_TRUSTED_PROXY_IPS', '*');` so that we know that ngrok is trust-worthy and allowed
-to pass http traffic as https.
+protected via https, however you will need to ensure you set the `SS_TRUSTED_PROXY_IPS` environemnt variable in your
+`.env`, e.g. `SS_TRUSTED_PROXY_IPS="*"` so that we know that ngrok is trustworthy and allowed
+to pass non-https traffic as https.
```

Comment:
Weird that the linter didn't pick up on this.
```suggestion
protected via HTTPS, however you will need to ensure you set the `SS_TRUSTED_PROXY_IPS` environemnt variable in your
`.env`, e.g. `SS_TRUSTED_PROXY_IPS="*"` so that we know that ngrok is trustworthy and allowed
to pass HTTP traffic as HTTPS.
```
Not sure why you changed "HTTP" to "non-HTTPS"? "HTTP" is the specific protocol that gets passed though. Other protocols wouldn't be changed.